### PR TITLE
Add tutorials for c,mongodb,r,sql,swift

### PR DESCRIPTION
**Pull request by trajchevska**
Tutorials for C, R, MongoDB, SQL and Swift are added.

### CODE REVIEW

File: app/controllers/learning_materials_controller.rb [hunk 1]
```diff
@@ -23,4 +23,17 @@ def c_plus_plus
...
+
+  def c
+  end
+
+  def swift
+  end
+
+  def mongo_db
+  end
+
```
Comment (alma-castant) [inline app/controllers/learning_materials_controller.rb]: sql seems to be missing @trajchevska

**Comment by alma-castant** [comment]
Other than that it looks ready to merge. Leave a comment here when the commit has been amended ;)

**Comment by trajchevska** [comment]
@alma-castant I must have overwritten it with some of the other functions while testing :) It's fixed.

**Comment by alma-castant** [comment]
@DMips looks good to merge

**Pull request merged**